content/blog/2021-11-02-agent.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Introducing Prometheus Agent Mode, an Efficient Cloud Native Way for Forwarding Metrics 
+created_at: 2021-11-02
+kind: article
+author_name: Bartlomiej Plotka (@bwplotka)
+---
+
+> Bartek Płotka is a Prometheus Maintainer since 2019 and Principal Software Engineer at Red Hat. Co-author of the CNCF Thanos project. CNCF Ambassador and tech lead for the CNCF TAG Observability. In his free time he writes a book titled "Efficient Go" with O'Reilly. Opinions are my own!
+
+What I personally love in Prometheus project and one of the many reasons why I joined the team, was the laser focus on the project goals. Prometheus was always about moving boundaries when it comes to providing pragmatic, reliable, cheap, yet invaluable metric monitoring. Its ultra stable and robust APIs, querying language and integration protocols (e.g. Remote Write and [Open Metrics](https://openmetrics.io/)) allowed the CNCF metric ecosystem to grow on those strong foundations. This is why we don't see Kubernetes clusters without Prometheus running there. This is also why we can see community exporters for getting metrics about literally everything e.g. [containers](https://github.com/google/cadvisor), [eBPF](https://github.com/cloudflare/ebpf_exporter), [Minecraft server statistics](https://github.com/sladkoff/minecraft-prometheus-exporter) and even [plants health when gardening](https://megamorf.gitlab.io/2019/07/14/monitoring-plant-health-with-prometheus/). 
+
+The strong focus of the Prometheus community allowed other open source projects to grow (e.g. [Cortex](https://cortexmetrics.io/), [Thanos](https://thanos.io/) and more) to extend Prometheus data model beyond single clusters. Not mentioning cloud vendors adopting Prometheus API and data model (e.g. [Amazon Managed Prometheus](https://aws.amazon.com/prometheus/), [Google Cloud Managed Prometheus](https://cloud.google.com/stackdriver/docs/managed-prometheus), [Grafana Cloud](https://grafana.com/products/cloud/) and more). If you are looking for a single reason why Prometheus project is so successful, it is this: *Focusing the monitoring community on what matters*.
+
+In this blog post I would love to introduce you a new mode of running Prometheus called "agent". It is built directly into the Prometheus binary. The agent mode blocks some of the Prometheus features and optimizes the binary for scraping and remote writing to remote location. While reducing number of features on the solution sounds like a regression, I will explain you why it is a game changer for certain deployments in the CNCF ecosystem. I am personally super excited for this! (:
```

Comment:
I would avoid the word regression, and turn this positively.

Reducing number of features enables new usage patterns. It is a game changer for certain deployments in the CNCF ecosystem, as I will explain below.

Reply:
I called "regression" on purpose so it sounds negatively - so I can turn explain that it is opposite. Happy to remove it if still a problem.